ruby-on-rails-v6/README.md

```diff
@@ -0,0 +1,17 @@
+# README
+
+## Setup TradingView
+
+- Copy `charting_library/charting_library.js` into `app/javascript/packs/charting_library/charting_library.js`
+- Copy `datafeeds/udf/dist/*.js` into `app/javascript/packs/datafeeds/`
+- Copy `charting_library/*.html` into `public/charting_library/`
+- Copy `charting_library/bundles` into `public/charting_library/bundles`
```

Comment:
It seems that we have to run `yarn` before run the app?

Reply:
Also, is it possible to migrate to `npm` instead?

Reply:
Of course